Kmozart/free-mentors
# #167783511 Finishes setting up pivotal tracker integrations

#### What does this PR do?
- Finishes setting up pivotal tracker - GitHub integrations.
 
#### Description of Task to be completed?
- Linking pivotal tracker IDs to their corresponding GitHub branches.

#### How should this be manually tested?
- ```git-clone``` this repo, 
- cd into any branch and make any relevant changes to it.
- ```git push``` changes and then view from GitHub webhooks the pivotal-tracker integration. 
- view the latest pushes and see if their status is green in color.

#### Any background context you want to provide?
- Pivotal-tracker helps keep your work on track and in check, letting you know what are the relevant branches and ID features, chores or bugs you are working on.

#### What are the relevant pivotal tracker stories?
- ch-integrating-pivotal-tracker-with-GitHub-167783511

#### Screenshots (if appropriate)
![snipped](https://user-images.githubusercontent.com/48585401/62704217-83be7780-b9f3-11e9-9f16-61eb957d2a30.PNG)


#### Questions:
- N/A
